### PR TITLE
Fix Az.Network Syntax Errors

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Fix Az.Network Syntax Errors

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/help/Set-AzNetworkProfile.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ $networkProfile = Get-AzNetworkProfile -Name np1 -ResourceGroupName rg1
 
 $networkProfile.Tags = "TestTag"
 
-$networkProfile.ContainerNetworkInterfaceConfigurations = New-AzNetworkProfileContainerNicConfig -Name cnicconfig1
```
Comment (ziyuezh576) [inline src/Network/Network/help/Set-AzNetworkProfile.md:31]: New-AzContainerNicConfig

File: src/Network/Network/help/Set-AzServiceEndpointPolicy.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ The **Set-AzServiceEndpointPolicy** cmdlet create a service endpoint policy.
 
 ### Example 1: Sets a service endpoint policy
 ```powershell
-$serviceEndpointPolicy = Set-AzServiceEndpointPolicy -Name "Policy1" -ServiceEndpointPolicy $serviceEndpointPolicy -ResourceGroup "resourcegroup1"
+$serviceEndpointPolicy = New-AzServiceEndpointPolicy -Name "Policy1" -ResourceGroupName "resourcegroup1" -Location "eastus"
+Set-AzServiceEndpointPolicy -ServiceEndpointPolicy $serviceEndpointPolicy
```
Comment (ziyuezh576) [inline src/Network/Network/help/Set-AzServiceEndpointPolicy.md:28]: ?

File: src/Network/Network/help/Update-AzVpnGatewayNatRule.md [hunk 3]
```diff
@@ -47,10 +47,10 @@ The **Update-AzVpnGatewayNatRule** cmdlet updates a NAT rule associated with Vpn
 New-AzResourceGroup -Location "West US" -Name "testRG"
 $virtualWan = New-AzVirtualWan -ResourceGroupName testRG -Name myVirtualWAN -Location "West US"
 $virtualHub = New-AzVirtualHub -VirtualWan $virtualWan -ResourceGroupName "testRG" -Name "westushub" -AddressPrefix "10.0.0.1/24"
-New-AzVpnGateway -ResourceGroupName "testRG" -Name "testvpngw" -VirtualHubId $virtualHub.Id -BGPPeeringWeight 10 -VpnGatewayScaleUnit 2
+New-AzVpnGateway -ResourceGroupName "testRG" -Name "testvpngw" -VirtualHubId $virtualHub.Id -VpnGatewayScaleUnit 2
 $vpnGateway = Get-AzVpnGateway -ResourceGroupName "testRG" -Name "testvpngw"
 New-AzVpnGatewayNatRule -ResourceGroupName $vpnGateway.ResourceGroupName -ParentResourceName $vpnGateway.Name -Name "testNatRule" -Type Static -Mode EgressSnat -InternalMapping "10.0.0.1/26" -ExternalMapping "192.168.0.0/26"
-$natRule = Update-AzVpnGatewayNatRule -InputObject $vpnConnection -IpSecPolicy $ipsecPolicy
+$natRule = Update-AzVpnGatewayNatRule -InputObject $vpnConnection
```
Comment (ziyuezh576) [inline src/Network/Network/help/Update-AzVpnGatewayNatRule.md:53]: ?

File: src/Network/Network/help/Update-AzVirtualHub.md [hunk 4]
```diff
@@ -46,8 +46,8 @@ The **Update-AzVirtualHub** cmdlet updates a virtual hub.
 ```powershell
 New-AzResourceGroup -Location "West US" -Name "testRG"
 $virtualWan = New-AzVirtualWan -ResourceGroupName "testRG" -Name "myVirtualWAN" -Location "West US"
-New-AzVirtualHub -VirtualWan $virtualWan -ResourceGroupName "testRG" -Name "westushub" -AddressPrefix "10.0.1.0/24"
-Update-AzVirtualHub -VirtualWan $virtualWan -ResourceGroupName "testRG" -Name "westushub" -AddressPrefix "10.0.2.0/24"
+$virtualHub = New-AzVirtualHub -VirtualWan $virtualWan -ResourceGroupName "testRG" -Name "westushub" -AddressPrefix "10.0.1.0/24"
+Update-AzVirtualHub -InputObject $virtualHub -AddressPrefix "10.0.2.0/24"
```
Comment (ziyuezh576) [inline src/Network/Network/help/Update-AzVirtualHub.md:50]: ?

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

**Review by ziyuezh576 — APPROVED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**